libsql-server/tests/cluster/replication.rs

```diff
@@ -1,10 +1,30 @@
-use std::sync::Arc;
+use std::sync::atomic::AtomicBool;
+use std::sync::atomic::Ordering::Relaxed;
 use std::time::Duration;
+use std::{io::Read, sync::Arc};
 
+use axum::ServiceExt;
+use bytes::Bytes;
+use futures::never;
+use futures_core::stream::BoxStream;
 use insta::assert_debug_snapshot;
 use libsql::Database;
-use libsql_server::config::{AdminApiConfig, DbConfig, RpcClientConfig, RpcServerConfig};
+use libsql_replication::{
+    frame::{FrameBorrowed, FrameMut},
+    rpc::replication::{Frame, HelloRequest, HelloResponse, LogOffset},
+};
+use libsql_server::{
+    config::{AdminApiConfig, DbConfig, RpcClientConfig, RpcServerConfig},
```

Comment:
wait, all those headers were missing? Or is it the IDE getting too eager with fixing dependencies?